### DISCUSSION TITLE
“Not Ready” - Unable to reach ‘rs-ny.rustdesk.com’ from my public ip.

### DISCUSSION THREAD

**Discussion by VegethB** [rustdesk/rustdesk#9691] [General]
### Bug Description

For the past month, the official rustdesk server has become unreachable. It had been running for 1 year without any problems.
With a `tracert` it tells me that it “dies” after the Amsterdam hop.
The network has not changed, no devices have been added, and the public ip has been the same for 20 years. There are no antivirus or firewalls in between.

I just don't know what to do. 
Rustdesk works locally and/or with a self-hosted server.... Unfortunately, however, we need to connect through the public rs.ny.rustdesk.com.

Thanks in advance

### How to Reproduce

.

### Expected Behavior

Connecting to the pubblic server

### Operating system(s) on local side and remote side

All

### RustDesk Version(s) on local side and remote side

From 1.2.6 to Latest

### Screenshots

![image](https://github.com/user-attachments/assets/c42f2686-22d0-4538-b931-432e3104c17c)

**Comment by rustdesk** [comment]
https://www.reddit.com/r/rustdesk/comments/1cr8kfv/should_you_selfhost_a_rustdesk_server/
